developer-docs-site/docs/guides/handle-aptos-errors.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Handling Errors in Aptos and Move"
+slug: "handle-aptos-errors"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Handling Errors in Aptos and Move
+
+This page catalogs common errors encountered in the Aptos blockchain and explains how to resolve them wherever possible. As with all software, the code itself is the source of truth for error handling and will always contain entries not found here.
+
+Instead, this matrix aims to help you address those errors most typically found, misunderstood, or both.
+
+For the sources of these errors, see:
+
+  * [vm_status.rs](https://github.com/move-language/move/blob/main/language/move-core/types/src/vm_status.rs)
+  * [error.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/move-stdlib/sources/error.move)
+  * [account.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/account.move)
+  * [coin.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/coin.move)
+  * [token.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move)
+  * [token_transfers.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token_transfers.move)
+
+Of course, many more errors exist in many more files. Help us update this list by sending pull requests containing the errors you encounter. If you don't know how to resolve the error, as described int the *Action* column, simply leave it blank.
```

Comment:
Suggest remove the first sentence, "Of course..." It feels a bit too colloquial.

Reply:
Will do.

Reply:
Done. Thanks.